run_test.sh

```diff
@@ -74,6 +74,8 @@ git clone https://github.com/AMReX-Codes/amrex.git
 cd amrex && git checkout --detach 1ffc6e427e3aea9302602190c8fc1cfbea0c9725 && cd -
 # warpx-data contains various required data sets
 git clone --depth 1 https://github.com/ECP-WarpX/warpx-data.git
+# openPMD-example-datasets contains various required data sets
+git clone --depth 1 --branch draft https://github.com/openPMD/openPMD-example-datasets
```

Comment:
Could you instead use `wget` or `curl` to download only the two files that you need from this repo?
Otherwise, we might spend a lot of time downloading large datasets (from `openPMD-example-datasets`) that are not actually used in the test.

Reply:
An earlier version of the PR as written by Yin-Yinjian Zhao did that, and I could never get it to actually run successfully in the automated CI environment. If there's a working example, I'd be happy to try following it

Reply:
ping @RemiLehe

Reply:
Phil, here is an example:
https://github.com/openPMD/openPMD-api/blob/dev/share/openPMD/download_samples.sh

Reply:
ping @PhilMiller to finalize :)

Reply:
```suggestion
curl -sOL https://github.com/openPMD/openPMD-example-datasets/raw/4ba1d257c5b4897c0a3cd57742bb0987343a902e/example-femm-thetaMode.h5
curl -sOL https://github.com/openPMD/openPMD-example-datasets/raw/4ba1d257c5b4897c0a3cd57742bb0987343a902e/example-femm-3d.h5
```